srcpkgs/mongoose/template

```diff
@@ -1,29 +1,40 @@
 # Template file for 'mongoose'
 pkgname=mongoose
-version=6.18
-revision=4
+version=7.6
+revision=1
 build_style=gnu-makefile
-make_use_env=compliant
-makedepends="openssl-devel"
-short_desc="Easy to use Web server"
-maintainer="Orphaned <orphan@voidlinux.org>"
+makedepends="$(vopt_if openssl openssl-devel mbedtls-devel)"
+short_desc="Embedded Networking Library with TCP,UDP,HTTP,MQTT,WebSocket and SSL/TLS"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="GPL-2.0-only"
-homepage="https://cesanta.com/"
+homepage="https://mongoose.ws/"
 distfiles="https://github.com/cesanta/mongoose/archive/${version}.tar.gz"
-checksum=f5c10346abc9c72f7cac7885d853ca064fb09aad57580433941a8fd7a3543769
+checksum=1ef09d971b6de1a6317c109980d6fb5a9c19b39efef2506d6b76869644b3dafa
+
+make_use_env=yes
+make_build_target=linux-libs
+make_check_target=test
+make_check=no # CBA for now
+
+build_options=openssl
+desc_option_openssl="Use OpenSSL instead of MbedTLS for SSL"
+export SSL="$(vopt_if openssl OPENSSL MBEDTLS)"
 
 mongoose-devel_package() {
 	depends="${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/include
+		vmove "usr/lib/*.a"
 		vmove "usr/lib/*.so"
 	}
 }
 
 mongoose-doc_package() {
 	short_desc+=" - documentation"
 	pkg_install() {
-		vmove usr/share
+		vmkdir usr/share/mongoose
+		vcopy docs usr/share/mongoose
+		vcopy examples usr/share/mongoose
```

Comment:
Did the installation process change so that, after installation, `usr/share` contains no docs?

Reply:
Previously, there was no Makefile at all!  
We hand-rolled our own, and it did the same steps.